include/deal.II/lac/constraint_matrix.h

```diff
@@ -1203,6 +1213,37 @@ class ConstraintMatrix : public Subscriptor
   const LineRange get_lines() const;
 
 
+  /**
+   * Check if the current object is consistent on all processors
+   * in a distributed computation.
+   *
+   * This method checks if all processors agree on the constraints for their
+   * local lines as returned by get_local_lines(). This method is a collective
```

Comment:
Aren't we only checking the `locally_active_dofs` and not all from `get_local_lines` as said in the next paragraph and as we discussed? Would you mind adapting the documentation?